tests/init.pp

```diff
@@ -1 +1,4 @@
-class { 'homebrew': }
+class { 'homebrew':
+  user  => 'root',
```

Comment:
It works on Travis, yes, but often CI systems do some weird things to make root-only access work. I have not tested whether the root user is useable on a standard system, but Homebrew itself advises installing/using it as a non-root user.